packages/server/tests/helpers/prerequisiteDappTests.ts

```diff
@@ -0,0 +1,35 @@
+import app from '../../src/server';
+import shell from 'shelljs';
+import dotenv from 'dotenv';
+import path from 'path';
+
+dotenv.config({ path: path.resolve(__dirname, '../../../.env') });
+
+const USE_LOCAL_NODE = process.env.LOCAL_NODE || 'true';
+const LOCAL_RELAY_URL = 'http://localhost:7546';
+const RELAY_URL = process.env.E2E_RELAY_HOST || LOCAL_RELAY_URL;
+
+(function () {
+  if (USE_LOCAL_NODE) {
+    // set env variables for docker images until local-node is updated
+    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.30.0-alpha.0';
```

Comment:
Can drop alpha

Reply:
Version above 0.30.0-alpha.0 is not working with the dapp tests. This workaround will be removed, when this issue is resolved in local node.

Reply:
Move the DApp tests workaround to a separate small PR. We can get that in easily and rebase this one to unblock

Reply:
Done